procedure.md

```diff
@@ -355,7 +355,7 @@ Records of the evaluation specifics could include any of the following:
 
 **Reminder:** In the majority of situations, using this methodology alone does not result in [WCAG 2 conformance claims](https://www.w3.org/TR/WCAG22/#conformance-claims) for the target digital product; see [Relation to WCAG 2 Conformance Claims](#context) for more background.
 
-Product owners may wish to make public statements about the outcomes from evaluations following this methodology. This can be done when at least every non-optional methodology requirement is satisfied, the conformance target defined in [Step 1.2. Define the Conformance Target](#step1b) is satisfied by all samples audited (in [Step 4: Audit the Selected Sample](#step4)), and the product owner commits to ensuring the validity and maintaining the accuracy of the evaluation statement made.
+Product owners may wish to make public statements about the outcomes from evaluations following this methodology. This can be done when at least every non-optional methodology requirement is satisfied, the conformance target defined in [Step 1.2. Define the Conformance Target](#step1b) is satisfied by all samples evaluated (in [Step 4: Evaluatethe Selected Sample](#step4)), and the product owner commits to ensuring the validity and maintaining the accuracy of the evaluation statement made.
```

Comment:
```suggestion
Product owners may wish to make public statements about the outcomes from evaluations following this methodology. This can be done when at least every non-optional methodology requirement is satisfied, the conformance target defined in [Step 1.2. Define the Conformance Target](#step1b) is satisfied by all samples evaluated (in [Step 4: Evaluate the Selected Sample](#step4)), and the product owner commits to ensuring the validity and maintaining the accuracy of the evaluation statement made.
```